### ISSUE TITLE
Fixing Display of View Btn in research paper uploadation

### ISSUE THREAD

**Issue by Harshdev098** [Harshdev098/Research-Nexas#3]
## Current behaviour
After Clicking to the View btn in upload_file.html  the view should be hidded and not be displayed while displaying the uploaded papers

## Expected
The View Btn should not be displayed while showing the uploaded papers

## Screenshots
![Screenshot from 2024-10-01 01-51-13](https://github.com/user-attachments/assets/2eba642a-1891-4ead-806c-f2a441bd0f0f)

**Comment by Charul00** [comment]
can i work on this please assign me

**Comment by meetbhalodi11** [comment]
@Harshdev098 I can help here. please assign it to me

**Comment by meetbhalodi11** [comment]
@Harshdev098 can you please add the **hacktoberfest** tab here

**Comment by meetbhalodi11** [comment]
@Harshdev098 can you please reassign this issue to someone else? Thanks

**Comment by Harshdev098** [comment]
@Charul00 You can work now, Please go ahead!

**Comment by Charul00** [comment]
bro where is the routes file
routes where is assign 
please give me a specific backend file 
enpoint route where it is ?

**Comment by Harshdev098** [comment]
The main backend file where all the endpoints are - dbServer.js

**Comment by harish08102004** [comment]
Hey @Harshdev098 I can work on this.

**Comment by harish08102004** [comment]
Hey @Harshdev098 can you add the level label also.

**Comment by harish08102004** [comment]
Hey @Harshdev098  it's "Level 3" not label.

**Comment by harish08102004** [comment]
Hey @Harshdev098 I'm facing some issues while starting the server.
![image](https://github.com/user-attachments/assets/01bf12fb-eb9b-4bb6-a599-8cd56838c537)

**Comment by Harshdev098** [comment]
@harish08102004 View this [reference](https://github.com/Harshdev098/Research-Nexas/discussions/32) for this error. Update me if it does not solve your query

**Comment by harish08102004** [comment]
Hey I'm still facing the same issues.

**Comment by Harshdev098** [comment]
Update the mysql with mysql2  `npm update mysql2` and also take care of password in .env

**Comment by harish08102004** [comment]
I tried updating the mySql but still the same error.

**Comment by Harshdev098** [comment]
Have you updated it to mysql2 because updating it to mysql2 have solved queries for maximum peoples

**Comment by harish08102004** [comment]
ok let me try again and let you know

**Comment by harish08102004** [comment]
Hey this error is resolved for now.

**Comment by Harshdev098** [comment]
Is this related to the mysql upgradation or any other

**Comment by harish08102004** [comment]
Nothing to do with it.
One of the contributor uploaded their db name and password in the code instead of .env that's the problem.
![image](https://github.com/user-attachments/assets/027fee5f-e28a-4634-9072-222ccb2fa5d4)

**Comment by harish08102004** [comment]
I changed it couple of files. now it's working fine.

**Comment by harish08102004** [comment]
Please let me know once you created an issue for this.

**Comment by Harshdev098** [comment]
Sure
